dev-python/gitdb2/gitdb2-2.0.2.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} )
+
+inherit distutils-r1
+
+DESCRIPTION="GitDB is a pure-Python git object database"
+HOMEPAGE="
+	https://github.com/gitpython-developers/gitdb
+	https://pypi.python.org/pypi/gitdb2"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="
+	!dev-python/gitdb[${PYTHON_USEDEP}]
+	>=dev-python/smmap2-2.0.1[${PYTHON_USEDEP}]"
+DEPEND="${RDEPEND}"
+# Testsuite require files from a git repo
```

Comment:
Thanks for diging into that. Played around with it outside the ebuild:

```bash
mkdir /tmp/repo
cd /tmp/repo
git init
for ((i=0; i<2500;i++)); do touch file${i}; git add file${i}; done
git commit -m "Add files"
git clone --bare repo
cd /tmp/gitdb-2.0.2
TRAVIS=1 GITDB_TEST_GIT_REPO_BASE=/tmp/repo.git nosetests
```

However, the last test still fails as before:

```
F.....SSSSSS...S........
======================================================================
FAIL: test_reading (gitdb.test.db.test_git.TestGitDB)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/gitdb-2.0.2/gitdb/test/db/test_git.py", line 29, in test_reading
    assert gdb.size() >= ni
AssertionError

----------------------------------------------------------------------
Ran 24 tests in 0.848s

FAILED (SKIP=7, failures=1)
```